app/assets/stylesheets/themes/default/_participant.scss

```diff
@@ -0,0 +1,146 @@
+.participant-content {
```

Comment:
The block is not `-content` but `pariticpant-content`, so I prevented from inheriting. More important to no repetition?